contributing/giving-constructive-criticism.rst

```diff
@@ -0,0 +1,179 @@
+Giving constructive criticism
+=============================
+
+:doc:`Reviewing issues and pull requests </contributing/community/reviews>`
+is a great way to get started with contributing to the Symfony community.
+Anyone can do it! But before you give a comment, take a step back and think,
+is what you are about to say actually what you intend?
+
+Communicating over the Internet with nothing but text can pose a
+big challenge, especially if you remember that the Symfony community
+is world-wide and is composed of a wide variety of people with differing
+ideas and opinions.
+
+Not everyone speaks English or is able to use a keyboard. Some might
+have Dyslexia or similar conditions that affect their writing.
+
+Not to mention that some might have a bad experience from previous
+contributions (to other projects).
+
+You're not alone in this. This guide will try to help you write
+constructive, respectful and helpful reviews and replies.
+
+.. tip::
+
+    This guide is not about lecturing you to "conform" or give-up
+    your ideas and opinions but helping you to better communicate,
+    prevent possible confusion, and keeping the Symfony community a
+    welcoming place for everyone. **You are free to disagree with
+    someone's opinions, just don't be disrespectful.**
+
+First of, accept that many programming decisions are opinions.
+Discuss trade offs, which you prefer, and reach a resolution quickly.
+It's not about being right or wrong, but using what works.
+
+Tone of voice
+-------------
+
+We don't expect you to be completely formal, or to even write error-free
+English. Just remember one thing: Don't swear, and be respectful to others.
+
+Don't reply in anger or with an aggressive tone. You're angry, we understand
+that, but swearing/cursing and name calling doesn't really encourage anyone to
+help you. Take a deep breath, count to 10 and try to *clearly* explain what problems
+you encounter.
+
+Gender pronouns
+---------------
+
+While not "formally" required it's better to use gender-neutral pronouns.
+Unless someone "indicated" their pronouns, use "they", "them" instead of
+"he", "she", "his", "hers", "his/hers", "he/she", etc.
+
+Try to avoid using wording that may be considered excluding by non-native speakers,
+like "hey guys", which is in itself gender-neutral, but may be interpreted as exclusively
+male by people from other cultures or with other native languages.
```

Comment:
I think the issue is not necessarily one of accidental confusion but more that there is an effort underway to change the english language to note base words on the male version as stated in my now outdated comment.

Reply:
Do you have any suggestions for improvement? or is this fine as it is (for) now.

Reply:
https://github.com/symfony/symfony-docs/pull/9034#discussion_r161370891

Reply:
@sstok Here is my proposal, partially inspired by @curry684’s comments.

```Try to avoid using wording that may be considered excluding and needlessly gendered, like for example words that have a male base. For example we recommend to use other words like “folks”, “team”, “everyone” in place of “guys”.```

Reply:
Fine with me :+1: